addon/lib/document.js

```diff
@@ -116,6 +116,7 @@ export default Base.extend({
    */
   jexl: computed(function () {
     const documentJexl = new jexl.Jexl();
+    const onlyNumbers = (nums) => nums.filter((num) => !isNaN(num));
```

Comment:
Take this out of this property to the top of the file. This way the function is redefined in each computation of this property..